packages/sn-universal-header/src/components/header.tsx

```diff
@@ -0,0 +1,55 @@
+import { AppBar, createStyles, makeStyles, Toolbar } from '@material-ui/core'
+import React from 'react'
+import { HamburgerMenu, HamburgerMenuClassKey } from './hamburger-menu'
+import { ThreeDotMenu, ThreeDotMenuClassKey } from './three-dot-menu'
+
+export const HEADER_HEIGHT = '64px'
+
+const useStyles = makeStyles(() => {
+  return createStyles({
+    appBar: {
+      height: HEADER_HEIGHT,
+      backgroundColor: '#342cac',
+      boxShadow: 'none',
+    },
+    toolBar: {
+      display: 'flex',
+      justifyContent: 'space-between',
+      height: '100%',
+      paddingRight: 0,
+      paddingLeft: '20px',
+    },
+    hamburgerAndTitleContainer: {
+      display: 'flex',
+      alignItems: 'center',
+    },
+  })
+})
+
+type UniversalHeaderClassKey = Partial<ReturnType<typeof useStyles>>
+
+interface UniversalHeaderProps {
+  title: string
+  threeDotMenuItems?: React.ReactNode
+  classes?: UniversalHeaderClassKey
+  hamburgerMenuClasses?: HamburgerMenuClassKey
+  threeDotMenuClasses?: ThreeDotMenuClassKey
+}
+
+export const UniversalHeader: React.FunctionComponent<UniversalHeaderProps> = (props) => {
+  const classes = useStyles(props)
+
+  return (
+    <AppBar position="sticky" className={classes.appBar}>
+      <Toolbar className={classes.toolBar}>
+        <div className={classes.hamburgerAndTitleContainer}>
+          <HamburgerMenu classes={props.hamburgerMenuClasses} />
+          <div>{props.title}</div>
+        </div>
+        {props.threeDotMenuItems ? (
```

Comment:
props.threeDotMenuItems && props.threeDotMenuItems.length > 0

Reply:
it should be renamed to renderXY the type is React.Reactnode. Discussed